S2VX.Game/Editor/Containers/NotesTimeline.cs

```diff
@@ -20,10 +20,25 @@ public class NotesTimeline : CompositeDrawable {
         [Resolved]
         private S2VXStory Story { get; set; }
 
+        // Tick information is calculated by GetTickInfo for a given time
+        private struct TickInfo {
+            public double CurrentTick;
+            public double CurrentTickTime;
+            public int NearestTick;
+            public double NearestTickTime;
+            public bool OnTick;
+        }
+
+        public double NumTicks { get; private set; }
+        public double TimeBetweenTicks { get; private set; }
+        private int NumVisibleTicks { get; set; }
+        public int FirstVisibleTick { get; private set; }
+        public int LastVisibleTick { get; private set; }
```

Comment:
I think some of these properties can be private

Reply:
Cleaned up some unneeded variables